text/3455-t-test.md

```diff
@@ -0,0 +1,111 @@
+- Feature Name: n/a
+- Start Date: (2023-06-27)
+- RFC PR: [rust-lang/rfcs#3455](https://github.com/rust-lang/rfcs/pull/3455)
+- Rust Issue: n/a
+
+# Summary
+[summary]: #summary
+
+Create a new subteam focused on testing in the development workflow and to be responsible for
+ensuring Rust has a high quality automated testing experience that includes the capabilities
+developers expect when working with a modern programming language.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, the overall automated testing experience spans multiple components owned by different
+teams across the Rust Project (e.g. T-cargo with `cargo test` being the primary
+touch point for most users, T-libs for libtest, T-rustdoc for doctests, etc.).
+This makes it more difficult to establish a vision and coordinate efforts to achieve that vision.
+However, there isn't any single team focused on the testing picture holistically.
+Simultaneously, there are a number of well known needs and feature requests in the space that do not have an explicit
+owner driving the efforts to completion.
+
+For example, there's been some long standing requests to have additional test output
+formats, such as JSON ([#49359]) and JUnit, available on Stable Rust.
+While some of these are available as unstable features in Nightly Rust,
+in [Rust 1.70.0](https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html), a bug was fixed
+so unstable test features, like `cargo test -- --format json`, require using a nightly
+toolchain, like other unstable features. This caused breakage in certain editor/IDE and CI
+related tooling as they had been relying on the prior behavior to get test data
+programmatically, and little progress has been made in the last 5 years ([#49359]).
+
+Furthermore, there's been a growing interest in improving testing generally, like [better CI integration][ci] as well as requests for things like better support for custom test harnesses and frameworks (.e.g [#2318]).
+
+The new Testing team is intended to establish an overarching vision and provide focused attention on these areas. 
+
+[#49359]: https://github.com/rust-lang/rust/issues/49359
+[#50297]: https://github.com/rust-lang/rust/issues/50297
+[#2318]: https://github.com/rust-lang/rfcs/pull/2318
+[ci]: (https://internals.rust-lang.org/t/pre-rfc-implementing-test-binary-list-format-json-for-use-by-ide-test-explorers-runners/18308)
+
```

Comment:
Will the test sub-team also take responsibility for `cargo bench`? 
I noticed we almost forgot to stabilize benchmark testing:

- [Pre-RFC: Stabilize \`#[bench]\`, \`Bencher\` and \`black\_box\` - language design - Rust Internals](https://internals.rust-lang.org/t/pre-rfc-stabilize-bench-bencher-and-black-box/4565/50)
- [When will benchmark testing be stable? - The Rust Programming Language Forum](https://users.rust-lang.org/t/when-will-benchmark-testing-be-stable/25482/5)

Reply:
Now that we have the team, I expect we'll be laying out our plan.  I personally expect benchmark support to be included as it is closely tied in with test support.

That said, I personally also see a lot of this being done by making custom test harnesses a first class citizen and doing the work on most of this outside of lib`test`